apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/preadj/McsClaimResponseTransformerV2.java

```diff
@@ -33,7 +32,11 @@
   private static final String METRIC_NAME =
       MetricRegistry.name(McsClaimResponseTransformerV2.class.getSimpleName(), "transform");
 
-  private static final Set<String> CANCELLED_STATUS_CODES = Set.of("r", "z", "9");
+  private static final Map<String, ClaimResponse.ClaimResponseStatus> STATUS_MAP =
```

Comment:
this name seems somewhat misleading since it can only map the statuses for cancelled things. further logic is required by the consumer of this to fully map the status.

Reply:
I see your point.  We are only using [2 of the 4 status codes](http://hl7.org/fhir/valueset-fm-status.html), "active" and "canceled", so from this logic if it's not in that map than it needs to be "active".  However, someone new to the project may wonder whether "draft" or "entered-in-error" should be used.

@KyRobbins could we maybe change the name of this Map and/or add some documentation to let the readers know we are only using the "active" and "cancelled" states to represent our data.